PermissionsEx/permissions.yml

```diff
@@ -2302,6 +2302,25 @@ groups:
         - essentials.skull
         - essentials.skull.*
         - essentials.gamemode.survival
+  doctordoom:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&2D&ao&2ct&ao&2r &aD&2o&ao&2m'
+      suffix: '&b'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
+        - ch.alias.hoveron
+        - ch.alias.hoveroff
+        - essentials.tp
+        - essentials.tp.*
+        - essentials.gamemode
+        - essentials.gamemode.creative
+        - essentials.skull
+        - essentials.skull.*
```

Comment:
http://blog.mysanantonio.com/geekspeak/files/2012/03/doctor_vs__doctor_by_docshaner-d36i641.jpg
He can create more of himself out of nothing